<p>Eukaryotic transcription is more complex mainly because of nuclesome that results in chromatin structures which is a general inhibitor for transcription needing activator to overcome this.</p>

<h1 id="nucleosome-remodelling-complexes">Nucleosome Remodelling Complexes</h1>
<p>Use ATP binding and hydrolysis to slide histone and remove or exchange histone octomers
from one DNA to another and they can also change histone sub-units.</p>

<p>They bind to histone octomer and they are double stranded DNA translocase. It is not yet
clear how histone exchange or removal is ahcieved by these complexes.</p>

<p>It is known H3 and H2 subunit of histone can be exchanged (various PDB avalible). Usually the seqeunce is different at the N or C terminus. Though the functional effect of such 
exchange is not well-understood. E.g. H2A.Z is often found in nucleosomes neighboring nucleosome-free regions.</p>

<h1 id="histone-modifying-enzymes">Histone Modifying Enzymes</h1>
<p>They add molecules to histone. Such as ubiquitin (100 res peptide similar size as histon)
or simply acetyl group. Usualy at (+ve group rich) N-terminal tails that pokes out of the DNA strands. 
Dynamic modification that can occur in reverse too.</p>

<p>Acetylation of lysine and methylation of lysine and arginine, they are Histone acetyltransferase or methyltransferase (HAT and HMT) and histone deacetylase or demethylase (HDAc HDMe). Such enzymes are histone subunit specific and site specific.</p>

<p>Acetylation would cause the lysine to become neutral, methylation locks the residue to
stay +ve charged. <!--Hence acetylation tend to make (-ve charged DNA) more accessible.-->
Heavily acetylation nucleosome cannot form higher order structures that further compact
the chromotin.</p>

<p>More commonly, modifications allow other proteins to recongnize the site. Such protein reader complexes
includes bromodomains for (acetylation), and many more for methylated sites (presumably becuase methylation do not have any direct effect on the nucleosome structure that acetylation has).</p>

<h1 id="nucleosome-positioning">Nucleosome Positioning</h1>
<p>Nucleosomes tend to not bind at G-C <em>OR</em> A-T rich regions. (DNA affect)</p>

<p>=================================================</p>

<p>L7</p>

<h1 id="rnap">RNAP</h1>

<p>There are three main RNA polymerases, I (rRNA) transcibes the most RNA but only one type - the ribosomal gene. II(mRNA, PDB 1I50) transcribes the least amount of RNA but makes the most abundant set of genes.</p>

<p>General transcription factors are proteins that bind to promoter sites on DNA</p>

<h2 id="enhancers">Enhancers</h2>
<p>Enhancers can be upstream, downstream, or within the transcribed region (generally in an intron). Enhancers need to be able to make an eRNA to be active. Enhancers encode eRNA which facilitate looping of DNA between the enhancer and promoter, bringing more factors to the promoter that promote transcription.</p>

<h2 id="insulators">Insulators</h2>

<p>===================</p>

<p>L8 
RNAP interactions with transcription factors</p>

<p>===================</p>

<p>L9</p>

<p>Phosphyrlation on the RNAP activates capping of the messenger RNA. 5’ end allows it to be stable, AAUAA sequence signals polyadenylation initiation (this is after the stop codon). a poly-A tail (~200 nucleotide) is then added to the end.</p>

<p>Phosphorylation of serine 5 of the RNA Pol II CTD recruits capping enzymes. This phosphylation occurs by the transcriptional initiation factor TFIIH. Capping enzymes are thereby targeted to the  5 prime end of nascent (newly made) mRNA. The 5’ cap protects mRNA from  exonuclease cleavage and is recognized by</p>

<p>with addition of AMP comes polyA binding protein which regulates the polyA addition length.</p>

<p>Once desired mRNA captured. The RNAP keeps transcribing and only falls off when an exonuclease (RAT1) completely degrades away the uncapped RNA which causes the DNA to reanneal.</p>

<p>#Gene Silencing
Not sequence specific involves SIR (silent information regulator) proteins</p>
